pkg/apis/logging/v1alpha1/conversion.go

```diff
@@ -0,0 +1,153 @@
+package v1alpha1
+
+import (
+  "fmt"
+
+  metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+  loggingv1 "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+)
+
+func (src *LogForwarding) ConvertTo(dst *loggingv1.ClusterLogForwarder) error {
```

Comment:
how do you feel about "TranslateTo" ?

Reply:
Conversion is close to k8s an api-machinery terminology when a resource is translated/transformed/whatever to a different form and/or version.